packages/remix-dev/vite/plugin.ts

```diff
@@ -666,11 +666,11 @@ export const remixVitePlugin: RemixVitePlugin = (options = {}) => {
           return;
         }
 
+        let referencesLiveReload =
```

Comment:
Was trying to think of a better name, but names like `mightReferenceLiveReload` don't really read well when negated: `!mightReferenceLiveReload`, so I think its fine. Maybe `coarseFilterForLiveReload` or something might explain better what this is doing conceptually.

Reply:
Chatted offline, going with `hasLiveReloadHints` for now.